libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -367,7 +402,7 @@ void AP_Baro::update(void)
             if (is_zero(ground_pressure) || isnan(ground_pressure) || isinf(ground_pressure)) {
                 sensors[i].ground_pressure = sensors[i].pressure;
             }
-            float altitude = get_altitude_difference(sensors[i].ground_pressure, sensors[i].pressure);
+            float altitude = get_altitude_difference(sensors[i].ground_pressure, sensors[i].pressure) + _alt_offset.get();
```

Comment:
being picky here but having the offset applied on it's own line (altitude += _alt_offset) makes this a bit easier to read